h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
+        model['is_moderator'] = True
```

Comment:
👍 I agree with Rob's feedback here. And add that, imo, apis that have the "it's false/not supported because it is missing" fields are harder to pick up and start working with in general for lots of reasons.

That said, consistency on that end is probably the most important thing. So, this should set the precedent for all fields going forward and we should make sure that all existing fields do the same - though I think that is the case.